src/js/components/Video.js

```diff
@@ -106,26 +125,60 @@ export default class Video extends Component {
         <RefreshIcon size={controlIconSize} /> :
           <PlayIcon size={controlIconSize} />));
 
+    var fullScreenButton;
+    if (this.props.allowFullScreen) {
+      fullScreenButton = (
+        <Button plain={true} onClick={this._onFullScreen}>
```

Comment:
can we add a11yTitle to this button and move the icon to props instead of children?

```
<Button plain={true} onClick={this._onFullScreen} icon={<ExpandIcon />} a11yTitle='Toggle FullScreen' />
```

Just make sure you localize the string for a11yTitle. See this: https://github.com/grommet/grommet/blob/master/src/js/components/SkipLinks.js#L107